ephios/core/views/event.py

```diff
@@ -316,3 +320,24 @@ def form_valid(self, form):
             CustomEventParticipantNotification.send(self.object, form.cleaned_data["mail_content"])
         messages.success(self.request, _("Notifications sent succesfully."))
         return redirect(self.object.get_absolute_url())
+
+
+class EventCalendarView(TemplateView):
+    template_name = "core/event_calendar.html"
+
+    def get_context_data(self, **kwargs):
+        year, month = kwargs["year"], kwargs["month"]
+        shifts = Shift.objects.filter(start_time__month=month, start_time__year=year)
+        calendar = ShiftCalendar(shifts, locale=get_language())
```

Comment:
On my machine, there is not locale `de` installed, just `de_DE`, so it errored away at first.

Reply:
ahhh i hate languages.
We are using `de` everywhere because this is broader than `de_DE` and your machine should choose `de_DE` anyway. We are also only compiling translations for `de`.

Reply:
Well we should have a test for it then.

Reply:
test for what exactly?

Reply:
The view